tests/e2e/entities/test_contract_deploy.py

```diff
@@ -237,7 +265,7 @@ def filtered_instantiate_contract_message_query(_filter):
                 self.assertNotEqual(transfer, [], "\nGQLError: No results returned from query")
                 self.assertEqual(transfer[0]["sender"], self.validator_address, "\nGQLError: sender address does not match")
                 self.assertEqual(transfer[0]["admin"], '', "\nGQLError: contract admin does not match")
-                self.assertEqual(int(transfer[0]["codeId"]), self._contract.code_id, "\nGQLError: contract code_id does not match")
+                self.assertEqual(int(transfer[0]["codeId"]), self.entities['instantiateMsg']['equal'][InstantiateMessageFields.code_id.value], "\nGQLError: contract code_id does not match")
```

Comment:
Can we add a comment to re-iterate the fact that the "first" contract is the one we're interacting with for most  assertions.